lib/qbwc/job.rb

```diff
@@ -83,7 +93,7 @@ def next
 
   def reset
     self.request_index = 0
-    self.requests = []
+    #self.requests = []
```

Comment:
I think this should still happen if `requests_provided_when_job_added` is false.